src/main/java/rx/Completable.java

```diff
@@ -1971,19 +1972,40 @@ public final void subscribe(CompletableSubscriber s) {
         } catch (NullPointerException ex) {
             throw ex;
         } catch (Throwable ex) {
+            Exceptions.throwIfFatal(ex);
             ERROR_HANDLER.handleError(ex);
```

Comment:
In observable, if the subscription fails, the `subscribe()` tries to call onNext on the subscriber. If the subscriber is unsubscribed, the original exception is sent to the plugin handler. If not but crashes further, a custom fatal exception is thrown. In Completable, we can't know if the completable subscriber is ready to receive the exception or not, thus the safe bet is to call the plugin handler and rethrow as NullPointerException. If nested, NPE is thrown as is further up without calling the handler again.